Library/Formula/gcsfuse.rb

```diff
@@ -0,0 +1,97 @@
+require "language/go"
+
+class Gcsfuse < Formula
+  desc "A user-space file system for interacting with Google Cloud Storage"
+  homepage "https://github.com/googlecloudplatform/gcsfuse"
+  url "https://github.com/GoogleCloudPlatform/gcsfuse/archive/v0.1.0.tar.gz"
+  sha256 "a65a1a70f75361f7ee9339199be98318d933b7986648860e179b5f24b130b156"
+
+  depends_on :osxfuse
+  depends_on "go" => :build
+
+  # Declare versioned Go dependencies
```

Comment:
We don't need comments on standard things like this - would you mind removing them? Thanks!